api/include/vehicles/kia_niro.h

```diff
@@ -0,0 +1,460 @@
+/**
```

Comment:
This file is the most significant change in this PR, and verification requires a test drive. I can run some calibration tests to ensure that everything works in shop, but since a test drive was performed I believe these values are generally reasonable.